src/integration-test/helpers/clients/idamClient.ts

```diff
@@ -31,6 +37,43 @@ export class IdamClient {
     })
   }
 
+  /**
+   * Authenticate user
+   *
+   * @param {string} email
+   * @param password the users password (optional, default will be used if none provided)
+   * @returns {Promise<string>}
+   */
+  static async authenticateUser (username: string, password: string = undefined): Promise<string> {
+
+    password = password || defaultPassword
```

Comment:
Please do not resign password argument. I preferred `${username}:${password || defaultPassword}`.

Reply:
done

[but this is typical JS solution for args with default value]